www/include/Administration/parameters/general/help.php

```diff
@@ -37,7 +37,10 @@
  * Sessions Properties
  */
 
-$help['tip_sessions_expiration_time'] = dgettext('help', 'Life duration of sessions.');
+$help['tip_sessions_expiration_time'] = dgettext(
+    'help',
+    'Life duration of sessions. The value in minutes cannot be greater than the value set in your php centreon.ini file.'
```

Comment:
It might be worth mentioning the exact PHP parameter (_session.gc_maxlifetime_).

Reply:
There's only four values. And only the admin of the server can modify those.
But why not. I'll modify the 15 files :'(

Reply:
done ;)